### ISSUE TITLE
jazzradio

### ISSUE THREAD

**Issue by hue4** [Feodor2/Mypal#200]
Can't get any noise out of it at all.  Worked previously.  Works on chrome. 

www.jazzradio.com

**Comment by lordpeluca** [comment]
I don't know exactly what you mean, but out of curiosity I went to the page you said and managed to hear the songs on it.

https://i.postimg.cc/KzhJB6HF/f1.jpg

**Comment by hue4** [comment]
> 
> 
> I don't know exactly what you mean, but out of curiosity I went to the page you said and managed to hear the songs on it.
> 
> https://i.postimg.cc/L8M5dvw5/f1.jpg
> 
> I'm using version 28.10.0

I wish somebody had told me there was a new MyPal version (which corrected the problem).  Automatic update did not work here!

**Comment by Feodor2** [comment]
In was a bug when autoupdate did not work and it is fixed now.

**Comment by lordpeluca** [comment]
Yes, I usually update everything manually (in the case of mypal I do it from GitHub), I usually leave the automatic update off, and every now and then I check in GitHub if a new update has come out.
